### DISCUSSION TITLE
$DROID token showing unknown

### DISCUSSION THREAD

**Discussion by datascience24** [phantom/docs#166] [Help & Support] [answered]
### Summary

Until about a week ago the $droid token was showing fine in the wallets with the name and picture. Now all of a sudden its showing as unknown token. 

Token address: GkywroLpkvYQc5dmFfd2RchVYycXZdaA5Uzix42iJdNo

### Steps to Reproduce

Just open wallet and looking at the $droid token in the token list

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment, marked as answer]
@datascience24 

It appears the token does not have the necessary metadata. Please have the metadata updated in accordance with the [Metaplex token standard.](https://docs.metaplex.com/programs/token-metadata/token-standard)
